<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_lager</class>
 <widget class="QWidget" name="Form_lager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_suche">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>xxx Buchungen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_suche">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>10</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tabellenfilter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_in">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Einen Wareneingang wird dann gebucht, wenn Ware von einem Lieferanten kommt und ins Lager eingelagert wird.Hier vorgenommene Buchungen haben Einfluss auf das Bestell-Modul&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Waren-Eingang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_out">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Einen Wareneingang wird dann gebucht, wenn Ware aus dem Lager entnommen wird.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Waren-Ausgang</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_artikel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inagain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eine Wieder-Einlagerung wird dann gebucht, wenn Ware z.B. aus dem Bankraum wieder ins Lager eingelagert wird.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Wieder-Einlagerung</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_korrektur">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eine Bestandskorrektur wird dann gebucht, wenn die Bestandsmenge eines Artikels im Programm nicht mit der Bestandsmenge im Regalfach Ã¼bereinstimmt, z.B. bei einer Inventur.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Bestandskorrektur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
